foyer-intrusive/src/eviction/mod.rs

```diff
@@ -12,55 +12,49 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-use crate::core::{adapter::Adapter, pointer::PointerOps};
-
+use crate::core::pointer::PointerOps;
 use std::fmt::Debug;
 
 pub trait Config = Send + Sync + 'static + Debug + Clone;
 
 pub trait EvictionPolicy: Send + Sync + 'static {
-    type Adapter: Adapter;
```

Comment:
It feels a little not unified here compared with other intrusive collections. Can we still use `Adapter` here?